<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="redbullCarPit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/RedBull.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ferrariCarPit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/Ferrari.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mercedesCarPit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/Mercedes.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mclarenCarPit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>20</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/McLaren.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mercedesCar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>490</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/Mercedes.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mclarenCar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>490</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/McLaren.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ferrariCar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>490</width>
      <height>106</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/Ferrari.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="redbullCar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>490</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/RedBull.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="redbullTimer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background:transparent;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ferrariTimer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background:transparent;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="mercedesTimer">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background:transparent;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="mclarenTimer">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background:transparent;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="standingsLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>340</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Fixedsys&quot;;
color: rgb(85, 255, 0);
background-color: rgb(67, 67, 67);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leaderboard">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/billboard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="grass">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>1151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/grass.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fence">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>1171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/fence.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dhlBoard">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>130</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/dhlBoard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dhlBoard_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/dhlBoard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dhlBoard_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/dhlBoard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dhlBoard_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/dhlBoard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="garageWall">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 189, 170)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="garageDoor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/garage.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="garageDoor_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>-10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/garage.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="garageDoor_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>-10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/garage.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="garageDoor_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>-10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/garage.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="finishLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="finishLine_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="finishLine_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="finishLine_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="finishLine_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <zorder>finishLine_5</zorder>
   <zorder>finishLine_4</zorder>
   <zorder>finishLine_3</zorder>
   <zorder>finishLine_2</zorder>
   <zorder>finishLine</zorder>
   <zorder>garageWall</zorder>
   <zorder>garageDoor</zorder>
   <zorder>garageDoor_2</zorder>
   <zorder>garageDoor_3</zorder>
   <zorder>garageDoor_4</zorder>
   <zorder>ferrariCarPit</zorder>
   <zorder>mclarenCarPit</zorder>
   <zorder>mercedesCarPit</zorder>
   <zorder>redbullCarPit</zorder>
   <zorder>fence</zorder>
   <zorder>dhlBoard_4</zorder>
   <zorder>dhlBoard_3</zorder>
   <zorder>dhlBoard_2</zorder>
   <zorder>dhlBoard</zorder>
   <zorder>grass</zorder>
   <zorder>mercedesCar</zorder>
   <zorder>mclarenCar</zorder>
   <zorder>ferrariCar</zorder>
   <zorder>redbullCar</zorder>
   <zorder>redbullTimer</zorder>
   <zorder>ferrariTimer</zorder>
   <zorder>mercedesTimer</zorder>
   <zorder>mclarenTimer</zorder>
   <zorder>leaderboard</zorder>
   <zorder>standingsLabel</zorder>
   <zorder>start</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
